src/app/resources/elements/edit-person/edit-person.element.html

```diff
@@ -1,13 +1,27 @@
 <template>
```

Comment:
Add a comment and link to the original implementation of the dialog html as this is a Bootstrap specific implementation for the dialog.

Reply:
Done